### DISCUSSION TITLE
Java,PHP,MySQL?

### DISCUSSION THREAD

**Discussion by R8BLOOD** [backend-br/forum#146] [Q&A]
Quais dessas 3 outras línguas vale a pena começar a estudar como primeira língua?

**Comment by brenoma** [comment]
Depende muito do que você quer, MySQL é um sistema gerenciador de banco de dados, que utiliza a linguagem SQL, que é uma linguagem voltada para banco de dados, eu recomendaria java para estudar programação orientada a objetos, como primeira linguagem talvez um Ruby, C, Python...

**Comment by Nanzitos** [comment]
Opá, um conselho, tenta fazer uma crud básico nas duas linguagens PHP/Java,  oque te agradar mais prologa os estudos.

**Comment by hebcodee** [comment]
Cara acho que Java com MySql

Mas depende bastante do problema que voce esta enfrentando
